UPGRADE-3.x.md

```diff
@@ -1,6 +1,14 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.25 to 3.26
+=========================
+
+### Commands
+
+Command classes were updated to inherit from `Command` instead of deprecated `ContainerAwareCommand`. Directly access to container will
```

Comment:
IMO, you have 2 choices to improve the 2nd sentence:
* >Direct access to container ...
* >Accessing directly to container ...